skyportal/tests/frontend/test_recent_sources.py

```diff
@@ -0,0 +1,54 @@
+import uuid
+from selenium.webdriver.common.action_chains import ActionChains
+
+from skyportal.tests import api
+
+
+def test_recent_sources(driver, user, public_source, public_group, upload_data_token):
```

Comment:
Could you please add a test where a user that is not part of the group and thus does not see the recently added source?